document/presentation/main.tex

```diff
@@ -280,8 +281,9 @@
   \item Add your ASIC folder in {\tt ./hardware/asic} using the other folders in there as examples
   \item In the file {\tt ./system.mk}:
   \begin{enumerate}
-    \item JTS: fill me
-    \item 
+     \item Add your ASIC node name in {\tt ASIC\_NODE}    
```

Comment:
Not consistent with sim and fpga -- use similar words; Define, not Add

Reply:
Not consistent with system.mk

ASIC_SERVER does not exist

Reply:
It's ASIC_COMPILE_SERVER, its not attached to any microcontroller , it just has the software to compile the ASIC